src/index.test.ts

```diff
@@ -21,14 +21,26 @@ describe('cli', () => {
                     client: 'sqlite3',
                     useNullAsDefault: true,
                 },
-                migrations: {},
+                migrations: {
+                    path: '/a/path',
+                },
             },
             commands,
         );
     });
 
-    it('command init is called in the cli constructor', () => {
-        expect(commands.init).toHaveBeenCalledTimes(1);
+    it('generates unique filenames over 1 second', done => {
+        const thing1: string = cli.filePath('thing1');
+        let thing2: string;
+        setTimeout(() => {
```

Comment:
This looks a bit bad - but the spec of the filePath command is to only be guaranteed unique >1 second